CineElorrietaMVC/src/main/java/Controlador/ControladorPanelResumen.java

```diff
@@ -0,0 +1,80 @@
+package Controlador;
+
+import Modelo.Modelo;
+import Vista.PanelGeneros;
+import Vista.PanelLogin;
+import Vista.PanelPeliculas;
+import Vista.PanelResumen;
+import Vista.Vista;
+
+public class ControladorPanelResumen {
+	@SuppressWarnings("unused")
+	private Modelo modelo;
+	private Vista vista;
+	private Controlador controlador;
+	private PanelResumen panelResumen;
+	
+	public ControladorPanelResumen(Modelo modelo, Vista vista, Controlador controlador) {
+		this.modelo = modelo;
+		this.vista = vista;
+		this.controlador = controlador;	
+	}
+	//Muestra las peliculas del sabado y domingo con su duraci�n
+	public void mostrarPanelResumen() {
+		this.panelResumen = new PanelResumen(this);
+		this.vista.mostrarPanel(this.panelResumen);
+
+		
+		
+		panelResumen.getTxtResumen().append(modelo.generar_resumen());
+	}
+		
+	/*	for(int i=0;i<ControladorPanelPeliculas.getPeliculasSabado().size();i++) {
+			
+			
+			
+			horas=(int) Math.floor((ControladorPanelPeliculas.getPeliculasSabado().get(i).getDuracion())/60);
+			minutos=(int) Math.round((ControladorPanelPeliculas.getPeliculasSabado().get(i).getDuracion())%60);
+			
+			panelResumen.getTxtResumen().append(ControladorPanelPeliculas.getPeliculasSabado().get(i).getTitulo()+"-> "+
+					horas+"h "+minutos+"m \n");
+			
+			
+		}
+		
+		panelResumen.getTxtResumen().append("\nDomingo: \n");
+		
+		for(int i=0;i<ControladorPanelPeliculas.getPeliculasDomingo().size();i++) {
+			horas=(int) Math.floor((ControladorPanelPeliculas.getPeliculasDomingo().get(i).getDuracion())/60);
+			minutos=(int) Math.round((ControladorPanelPeliculas.getPeliculasDomingo().get(i).getDuracion())%60);
+			
+			
+			panelResumen.getTxtResumen().append(ControladorPanelPeliculas.getPeliculasDomingo().get(i).getTitulo()+"-> "+
+					horas+"h "+minutos+"m \n");
+			
+			
+		}
+		
+	}*/
+	
+	public void accionadoBotonRechazarPanelResumen() {
+		controlador.navegarPanelGeneros();
+		ControladorPanelPeliculas.getPeliculasSabado().removeAll(ControladorPanelPeliculas.getPeliculasSabado());
+		ControladorPanelPeliculas.getPeliculasDomingo().removeAll(ControladorPanelPeliculas.getPeliculasDomingo());
```

Comment:
Probablemente resulte más interesante guardar las películas en un objeto nuevo que creéis en el modelo

Reply:
solucionado